### DISCUSSION TITLE
wrong name/brand for romaniana easybox - parcel locker ( is not property of eMAG)

### DISCUSSION THREAD

**Discussion by razor74** [openstreetmap/iD#9428] [Q&A]
Hi.
Please help with correcting brand for easybox , which is property of Sameday  - https://sameday.ro/easybox/ , not property of eMag. Any comercial company could send parcels with sameday easybox, not only eMag.

Thank you.

**Comment by tordans** [comment]
I suspect this is where you want to go:

![image](https://user-images.githubusercontent.com/111561/207687852-076b3359-ce4d-483b-a768-97dc874826a8.jpeg)

**Reply by razor74** [reply]
Is the first time when i post in github. I dont find that page with choosing template...

**Reply by tordans** [reply]
Welcome @razor74 and thanks for sharing what you found. The screen above is [on the "Issues"](https://github.com/openstreetmap/iD/issues) tab at ["New Issue"](https://github.com/openstreetmap/iD/issues/new/choose).

I understand that the discussions tab vs the issues tab is not easy do understand, unfortunately ([see my comment here](https://github.com/openstreetmap/iD/discussions/8841#discussioncomment-2280089)).
